graphql/queries/getExercises.ts

```diff
@@ -0,0 +1,31 @@
+import { gql } from '@apollo/client'
+
+const GET_EXERCISES = gql`
```

Comment:
I wasn't sure what to call this query. It does more than just get the exercises - it gets the lessons and alerts too which are used on the DOJO exercises page.

Reply:
`GET_DOJO_EXERCISES` maybe, but it's longer.

Reply:
Yeah, or maybe even something like `GET_DOJO_EXERCISES_PAGE_DATA`  since we're getting all the data we need for the DOJO exercises page.